src/app.js

```diff
@@ -1,7 +1,23 @@
 const express = require('express');
 
+const readerRouter = require('./routes/readerRouter');
+
+const bookRouter = require('./routes/bookRouter');
+
+const genreRouter = require('./routes/genreRouter');
+
+const authorRouter = require('./routes/authorRouter');
+
```

Comment:
Could condense these together:
```suggestion
const readerRouter = require('./routes/readerRouter');
const bookRouter = require('./routes/bookRouter');
const genreRouter = require('./routes/genreRouter');
const authorRouter = require('./routes/authorRouter');
```